framework/src/bundle/BundleManifest.cpp

```diff
@@ -165,7 +165,7 @@ namespace cppmicroservices
 
     BundleManifest::BundleManifest() : m_Headers(AnyMap::UNORDERED_MAP_CASEINSENSITIVE_KEYS) {}
 
-    BundleManifest::BundleManifest(AnyMap const& m) : m_Headers(m) {}
+    BundleManifest::BundleManifest(AnyMap m) : m_Headers(std::move(m)) {}
```

Comment:
I don't think this should be moved from based on the usage in BundlePrivate.cpp. This `BundleManifest` constructor is currently used in the `BundlePrivate::BundlePrivate(CoreBundleContext* coreCtx, std::shared_ptr<BundleArchive> const& ba)` constructor.